config/config.exs

```diff
@@ -25,3 +25,11 @@ config :logger, :console,
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env}.exs"
+
+config :addict,
+  secret_key: "243262243132244543362f4573334a324257346976794e2e7559764675",
```

Comment:
We should change this to a `System.get_env("SECRET_KEY")` call so we can override it in production.

Also consider moving it into the proper environment file (only moving the `from_email` and `secret_key`, the rest can stay here).